skins/cat17/src/app/lib/view_handler/fee_option_switcher.js

```diff
@@ -18,6 +18,7 @@ var objectAssign = require( 'object-assign' ),
 
 		update: function ( state ) {
 			_.each( this.elements, function ( feeOption ) {
+				// TODO no need to parseFloat when fee is converted to integer
 				var shouldBeDisabled = this.minimumFee[ state.addressType ] > feePerMonth( state.paymentIntervalInMonths, parseFloat( feeOption.val() ) ) ;
```

Comment:
This has to be touched when we change the amount values in HTML, right?

Reply:
This line needs the same conversion as in donationForm. Also, the membership fee template needs the same "multiply value by 100" shenanigans as in the `payment_preset_amounts.html.twig`. But since we don't have that page yet, there is no way to add a "TODO"